ifrs17/Utils/ImportCalculationMethods.ipynb

```diff
@@ -349,7 +349,7 @@
         {
             "cell_type": "code",
             "source": [
-                "public static async Task ExtendParsedVariables (this IWorkspace workspace, Systemorph.Vertex.Hierarchies.IHierarchicalDimensionCache hierarchyCache) {}"
+                "public static async Task ExtendParsedVariables (this IWorkspace workspace, Systemorph.Vertex.Hierarchies.IHierarchicalDimensionCache hierarchyCache, Dictionary<string, DataNodeData> dataNodeDataBySystemName) {}"
```

Comment:
can't you input the ParsingStorage instead of the hierarchy and the dict?

Reply:
The ParsingStorage class is defined in Importers notebook which is not imported in the ImportCalculationMethods, I would either need to move the definition of ParsingStorage class or import the Importers notebook to ImportCalculationMethods. I found that defining 2 parameters for the ExtendParsedVariables was the easiest and required less changes.